src/settings/MatchProfiles/ViewMatchProfile.js

```diff
@@ -1,26 +1,374 @@
-import React, { Component } from 'react';
+import React, {
+  Component,
+  Fragment,
+} from 'react';
 import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
 
-import { Pane } from '@folio/stripes/components';
+import {
+  Icon,
+  Pane,
+  Button,
+  Checkbox,
+  PaneMenu,
+  Headline,
+  KeyValue,
+  Accordion,
+  SearchField,
+  AccordionSet,
+  MultiColumnList,
+} from '@folio/stripes/components';
+import {
+  ViewMetaData,
+  SearchAndSortQuery,
+} from '@folio/stripes/smart-components';
+import {
+  AppIcon,
+  IntlConsumer,
+  TitleManager,
+  stripesConnect,
+} from '@folio/stripes/core';
+
+import { Preloader } from '../../components/Preloader';
+import { resultsFormatter } from '../JobProfiles';
+import {
+  createUrl,
+  sortStrings,
+  withCheckboxList,
+  checkboxListShape,
+} from '../../utils';
+import {
+  SYSTEM_USER_ID,
+  SYSTEM_USER_NAME,
+} from '../../utils/constants';
+
+import searchAndSortCss from '../../components/SearchAndSort/SearchAndSort.css';
+import sharedCss from '../../shared.css';
 
 // TODO: view component will be developed in UIDATIMP-140
```

Comment:
It seems, it should be deleted in the scope of UIDATIMP-140

Reply:
Done